src/services/fibery/query.js

```diff
@@ -67,11 +67,10 @@ export const TAGS = Object.entries(TAGS_LOCALIZED_FIELDS).reduce((acc, [key, fie
 }, []);
 
 const RELATED_ENT_FIELDS = {
-  Collection: mapAppNameLocales(['Name', 'Description'])
-    .concat(addAppName('Slug'))
-    .concat({
-      'user/Cover': IMAGE,
```

Comment:
@Drapegnik Maybe you have an idea how that should be fixed. If I don't comment and try to import an article with no collection from Fibery, I get a validation error on our backend, because `collection` object is not empty, because it is actually `{ "user/Cover": null }`.

Reply:
Added a dirty hack